lib/animina/chat_completion.ex

```diff
@@ -80,7 +80,7 @@ Please return the messages as follows:
 
     {:ok, task} =
       Ollama.completion(client,
-        model: "llama3.1:8b",
+        model: llama_version(),
```

Comment:
Are we going to use `llama_version/1` anywhere else in this code or is this the only place? I think it might be best to get rid of that function and just use

```elixir
model: Application.get_env(:animina, :llama_version)
```